base_layer/wallet_ffi/src/lib.rs

```diff
@@ -3455,7 +3457,27 @@ pub unsafe extern "C" fn wallet_add_base_node_peer(
         ptr::swap(error_out, &mut error as *mut c_int);
         return false;
     }
-
+    // Start once off Output manager validation protocols that need the base node peer to be set
+    if let Err(e) = (*wallet).runtime.block_on(
```

Comment:
These `block_on` might cause unresponsive UI. Just something to look out for

Reply:
In the FFI calls all calls through to the wallet use block_on because the FFI library calls needs to resolve before returning. That's how error states are passed back to the FFI client. It would be up to the client to manage these calls in an async way.